hr_contract_multi_jobs/models/hr_contract.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2014 Savoir-faire Linux. All Rights Reserved.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    contract_job_ids = fields.One2many('hr.contract.job',
```

Comment:
Please, break the line just after `(` in order to get a nicer and more readable indentation